docs/getting-started-on-snowplow-open-source/quick-start-aws/index.md

```diff
@@ -88,95 +96,78 @@ terraform plan
 terraform apply
 ```
 
-This will output your `iglu_server_dns_name`. Make a note of this, you'll need it when setting up your pipeline. If you have attached a custom ssl certificate and set up your own DNS records then you don't need this value.
+The deployment will **take roughly ~15 minutes to deploy** and once done will output your `iglu_server_dns_name`. Make a note of this, you'll need it when setting up your pipeline. If you have attached a custom SSL certificate and set up your own DNS records then you don't need this value.
 
 **Step 3 (optional): Add your schemas to Iglu Server**
```

Comment:
I would remove this from here. I don’t think anyone would write up schemas before deploying a pipeline. And it’s distracting when we mix installation with usage. We’ll point people to the usage doc later in the flow. (Also, it’s not a good idea to duplicate these instructions across the docs — we should have a [single place](https://docs.snowplow.io/docs/understanding-tracking-design/managing-your-data-structures/iglu/) where this is explained and just link to it.)